src/main/resources/org/gradle/profiler/report/report-template.html

```diff
@@ -272,6 +273,16 @@
         datasets: []
     },
     options: {
+        plugins: {
+            crosshair: {
+                zoom: {
+                    enabled: false
```

Comment:
Hmm..., there is now a "Reset Zoom" button, though this button seems to do nothing. At least I don't know how to zoom.

Reply:
Where do you see that button? :o It shouldn't be displayed at all.